docs/paper/paper.md

```diff
@@ -0,0 +1,132 @@
+quantum computers and simulators.
+It uses many other open source packages.
+These include Numpy [@Harris2020] for fast numerical computing, lmfit [@Newville2014] to fit complex 
+models to data, CVXPY [@Diamond2016] for convex optimization, Matplotlib [@Hunter2007] for plotting, 
+and uncertainties [@Lebigot2016] to provide measurements with a mean and a standard deviation.
+
+
+# Statement of need
+
+Quantum computing processes information following the laws of quantum mechanics.
+Quantum computers, like classical computers, must be programmed to perform quantum computations. 
+A quantum computer consists of qubits which store information in 
+quantum states, along with additional hardware elements, such as resonators to couple, control, 
+and readout the qubits. 
+The different elements in the quantum hardware have properties that must be characterized to 
+calibrate the quantum gates that process the information. 
+Furthermore, the quality of these gates has to be benchmarked to measure the overall performance 
+of the quantum computer. 
+This characterization and calibration requires an extensive set of experiments and analysis routines.
+
+Quantum development software packages such as Qiskit, ReCirq [@QuantumAI], tKet [@Sivarajah2021], and
+Forest [@Smith2016] are part of the quantum stack to execute quantum circuits on hardware.
+They also enable high-level applications that abstract away the quantum hardware. 
+Forest-benchmarking [@Forest] and pyGSTi [@pyGSTi] are tailored towards benchmarking of quantum hardware.
+However, there is still a need for open-source software that enables researchers and hardware 
+maintainers to easily execute characterization and calibration experiments.
+`Qiskit Experiments` is unique in this perspective as it provides low-level characterization 
+experiments that integrate with pulse-level control [@Alexander2020].
+In addition, `Qiskit Experiments` provides a calibration framework to manage device calibration.
+This framework is usable with any hardware exposed as a Qiskit backend.
+For example, the `Qiskit Experiments` framework has been used to explore measurements without qubit 
+reset [@Tornow2022], characterize positive operator value measures [@Fischer2022], quantum 
+states [@Hamilton2022], and time-evolutions [@Greenaway2022], as well as calibrate gates [@Vazquez2022].
+
+![
+Conceptual framework of Qiskit Experiments.
+The circuits are run as jobs on the quantum backends.
+If an experiment exceeds the maximum circuit limit per job it is broken down in multiple jobs.
+The raw data, figures and analysis results are contained in the `ExperimentData` class.
+\label{fig:framework}](framework.pdf){ width=50% }
+
+# Example usage
+
+Here, we exemplify `Qiskit Experiments` with a Quantum Volume (QV) measurement [@Cross2019].
+We execute random SU(4) circuits on a noisy simulator of a quantum backend to quantify
```

Comment:
```suggestion
We execute random SU(4) circuits on a simulator with a noise model of a quantum device to quantify
```

Reply:
I don't think we have to remove backend here because it's already used in multiple places, including the figure. Maybe we can change the first occurrence of it to be a bit clearer that it's physical hardware with a software interface.

Reply:
Yeah that makes sense. I think backend has [several meanings](https://en.wikipedia.org/wiki/Frontend_and_backend). In our case backend refers to a server in the client sever model.